### ISSUE TITLE
Is it possible to initialise an empty queryset? (e.g. Model.objects.none() in Django)

### ISSUE THREAD

**Issue by belthaZornv** [barseghyanartur/django-elasticsearch-dsl-drf#238]
I can't seem to find anything besides `EmptySearch` to initialise an empty queryset - Although `EmptySearch` doesn't have the complete set of properties et cetera required. 

I need to return an empty queryset in a view; did anyone face this problem?

**Comment by barseghyanartur** [comment]
The `EmptySearch` is supposed to have a complete set of properties you need. If it does not, I'm more than eager to accept a PR.

**Comment by barseghyanartur** [comment]
@belthaZornv:

It would be useful to at least know what your use case is and which things are missing.

**Comment by belthaZornv** [comment]
Hey @barseghyanartur! - we have a view using `DocumentViewSet` which then calls `filter_queryset` using a number of `backends`  and `django.core.paginator`; the `paginator` uses:

```
    def page(self, number):
        """Return a Page object for the given 1-based page number."""
        number = self.validate_number(number)
        bottom = (number - 1) * self.per_page
        top = bottom + self.per_page
        if top + self.orphans >= self.count:
            top = self.count
        return self._get_page(self.object_list[bottom:top], number, self)
```

Where `object_list` (EmptySearch) is not subscriptable (raises) 

This is the first issue I faced trying to make use of the `EmptySearch` class; I believe another issue was the `.execute()` - I've noticed a few other methods that are missing (in `Search` but not in `EmptySearch`) but those were the main issues.

**Comment by barseghyanartur** [comment]
@belthaZornv:

OK, slicing should be fixed now.

```python
from django_elasticsearch_dsl_drf.utils import EmptySearch
s = EmptySearch()
s[0:10]  # []
```

**Comment by belthaZornv** [comment]
Will test it out and will come back with issues and how to reproduce (if any) - thanks @barseghyanartur

**Comment by barseghyanartur** [comment]
Actually, just implemented a better thing. Check [this](https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/blob/master/examples/simple/search_indexes/viewsets/book/permissions.py#L358).

```python
class BookNoRecordsDocumentViewSet(BookPermissionsDocumentViewSet):
    def get_queryset(self):
        queryset = EmptySearch()
        queryset.model = self.document.Django.model
        return queryset
```

The output of request to http://localhost:8000/search/books-no-records/ would always be this:

```json
{
    "count": null,
    "next": null,
    "previous": null,
    "results": []
}
```

I'm adding up some other fixes and improfements here.

**Comment by belthaZornv** [comment]
Nice! will give it a go @barseghyanartur ! will let you know how it goes :grin:

**Comment by barseghyanartur** [comment]
Released in 0.22
